üõ† UNDER CONSTRUCTION

Eunmin Eddie Lee
===

Work Experience
---

**Quality Assurance** First Hawaiian Bank __Apr 2021 to present__ 

**Data Scientist(Image Deep Learning)** Natural Resource Data Solutions __Oct 2020 to Feb 2021__ 5 months

**Data Scientist(Bioinformatics)** Daegu Gyeongbuk Institute of Science and Technology  __Mar 2017 to Feb 2020__ 3 years

**Web Developer & Quality Assurance** Republic of Korea Air Force  __Mar 2017 to Feb 2020__ 1 year 9 months

**Research Assistant** Washington University in St. Louis  __Mar 2017 to Feb 2020__ 2 years 3 months

Technical Skills
---
Machine Learning,
Data Engineering,
Data Mining,
Data Analysis,
Data Visualization,
Data Science,
Big Data,
Artificial Intelligence (AI),
Computer Vision,
Software Development,
Python,
PyTorch,
R,
SQL,
Microsoft Project,
Matlab,
Mathematica,
Java,
C#,
.NET Framework,
C/C++,
JavaScript,
React.js,
Electron.js,
Redux.js,
HPCC,
Docker Products,
Azure DevOps,
Microsoft Azure,
Amazon Web Services (AWS),
SolidWorks,
AutoCAD,
Process Control,
Research,
Teaching,
NGS,
Single Cell RNA Sequencing



Publications:
---
‚ùñ	Yum, M. K., Han, S., FInk, J.,Wu S. S., Dabrowska, C., Trendafilova, T., Mustata, R., Chatzeli L., Azzarelli, R., Pshenichnaya I., **Lee, E.**, England F., Kim, J. K., Stange, D. E., Philpott A., Lee, J., Koo, B., Simons, B. D., Tracing oncogene-driven remodelling of the intestinal stem cell niche, Nature, 594, 442-447. (2021)

‚ùñ	Lee, M.\*, **Lee, E.\***, Han, S. K.\*, Choi, Y., Kwon, D., Choi, H., Lee, K., Park, E. S., Rha, M., Joo, D. J., Shin, E., Kim, S., Kim, J. K., Lee, Y. J., Transcriptional landscapes of innate T cells at the single cell level reveal a shared lineage differentiation process regulated by clonal selection and proliferation in the thymus. Nature Communications, 11 4367. (2020) 

‚ùñ	Noorani, I.\*, de la Rosa, J., Choi, Y., Strong, A., Ponstingl, H., Vijayabaskar, M. S., Lee, J., **Lee, E.**, Richard-Londt, A., Friedrich, M., Furlanetto, F., Fuente, R., Banerjee, R., Yang, F., Law, F., Watts, C., Rad, R., Vassiliou, G., Kim, J. K., Santarius, T., Brandner, S., Bradley, A.,  PiggyBac mutagenesis and exome sequencing identify genetic driver landscapes and potential therapeutic targets of EGFR-mutant gliomas. Genome Biology, 21 181. (2020) 

‚ùñ	Han, S.\*, Fink, J.\*, J√∂rg, D. J.\*, **Lee, E.**, Yum, M. K., Chatzeli, L., Merker, S.R., Josserand, M., Trendafilova, T., Andersson-Rolf, A., Dabrowska, C., Kim, H., Naumann, R., Lee, J. H., Sasaki, N., Mort, R. L., Basak, O., Clevers, H., Stange, D. E., Philpott, A., Kim, J. K., Simons, B. D., Koo, B. K., Defining the identity and dynamics of adult gastric isthmus stem cells. Cell Stem Cell, 25(3), 342-356. (2019) 

‚ùñ	Kim, B.\*, **Lee, E.\***, Kim, J. K. Analysis of technical and biological variability in Single-Cell RNA Sequencing. Methods in Molecular Biology, 1935, 25-43. (2019) 

‚ùñ	Lee, K., **Lee, E.**, Song, C., Janik, M. J. Density functional theory study of propane steam reforming on Rh-Ni bimetallic surface: Sulfur tolerance and scaling/Bro√∏nsted-Evans-Polanyi relations. Journal of Catalysis, 309, 248-259. (2014) * co-first authorship 

    * co-first authorship

Education
---
**Master of Science (Computer Science)**
	Georgia Institute of Technology, Atlanta, GA

**Bachelor of Science (Chemical Engineering)**
	Pennsylvania State University, University Park, PA